### PR TITLE
Fix `DisplayServer` overrides

### PR DESCRIPTION
**Pull request by AThousandShips**
Introduced by conflict between:
* https://github.com/godotengine/godot/pull/89588
* https://github.com/godotengine/godot/pull/90549
* https://github.com/godotengine/godot/pull/90268

### CODE REVIEW

File: servers/display_server_headless.h [hunk 1]
```diff
@@ -168,11 +168,10 @@ class DisplayServerHeadless : public DisplayServer {
 	Error file_dialog_with_options_show(const String &p_title, const String &p_current_directory, const String &p_root, const String &p_filename, bool p_show_hidden, FileDialogMode p_mode, const Vector<String> &p_filters, const TypedArray<Dictionary> &p_options, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
 
 	void release_rendering_thread() override {}
-	void make_rendering_thread() override {}
```
Comment (AThousandShips) [inline servers/display_server_headless.h:171]: I don't know where this override came from, it isn't in the main display server, and hasn't been for some time AFAIK

**Comment by AThousandShips** [comment]
Will rebase this on the `script` name one to ensure one builds correctly

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by AThousandShips** [comment]
Thank *you*!

**Pull request merged**